### PR TITLE
ImageCache: fix responsiveParametersPatterns

### LINKED ISSUE #422: Part of the article is not displayed
URL => https://www.adslzone.net/2017/01/09/intel-kaby-lake-procesadores-caracteristicas-precios/

Wallabag web => full article
RSS => full article
Wallabag app => just half article content (text and images)

### PR DESCRIPTION
**Pull request by di72nn**
Fixes #422.

### CODE REVIEW

File: app/src/main/java/fr/gaulupeau/apps/Poche/network/ImageCacheUtils.java [hunk 1]
```diff
@@ -38,9 +38,9 @@
             = Pattern.compile("<img[\\w\\W]*?src=\"([^\"]+?)\"[\\w\\W]*?>");
 
     private static final Pattern[] responsiveParametersPatterns = {
-            Pattern.compile("srcset=\".*\""),
-            Pattern.compile("sizes=\".*\""),
-            Pattern.compile("data-zoom-src=\".*\""),
+            Pattern.compile("srcset=\"[^\"]*?\""),
```
Comment (Strubbl) [inline app/src/main/java/fr/gaulupeau/apps/Poche/network/ImageCacheUtils.java]: Good catch. my regex was the greedy one which is wrong in this case. I used the wrong quantifier. `[^\"]*?` looks complicated on the first view until i understood, that this means all chars except double quotes. `.*?` should work for the whole pattern, too, shouldn't it? But i think your expression is more explicit telling to not catch double quotes.

Comment (di72nn) [inline app/src/main/java/fr/gaulupeau/apps/Poche/network/ImageCacheUtils.java]: Yeah, `.*?` should work too.

**Comment by ngosang** [comment]
When will these changes be in Google Play?

**Comment by tcitworld** [comment]
Soon.

**Pull request merged**